OpenSSL/crypto.py

```diff
@@ -2304,6 +2372,19 @@ def verify(cert, signature, data, digest):
         _raise_current_error()
 
 
+def verify_cert(store_ctx):
+    """
+    Verify a certificate in a context.
+
+    :param store_ctx: The :py:class:`X509StoreContext` to verify.
+    :raises: Error
+    """
+    store_ctx._init()
+    ret = _lib.X509_verify_cert(store_ctx._store_ctx)
+    store_ctx._cleanup()
```

Comment:
I think that this all makes good sense and applied these changes in the latest diff. I'll make a note on my todo list to think about a nice top level verify_certificate method because it does feel strange with the current OpenSSL API. :)